Stop using `const char*` in string concatenations

cdumez opened the pull request:
#### 1cf337e0a50aaff16acd37edb6b4edf0c09ced97
<pre>
Stop using `const char*` in string concatenations
<a href="https://bugs.webkit.org/show_bug.cgi?id=274297">https://bugs.webkit.org/show_bug.cgi?id=274297</a>

Reviewed by Darin Adler.

* Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml:
* Source/WTF/wtf/text/StringConcatenate.h:
* Source/WebCore/platform/mediastream/gstreamer/GStreamerAudioCaptureSource.cpp:
(WebCore::GStreamerAudioCaptureSource::create):
* Source/WebCore/platform/mediastream/gstreamer/GStreamerDisplayCaptureDeviceManager.cpp:
(WebCore::GStreamerDisplayCaptureDeviceManager::computeCaptureDevices):
(WebCore::GStreamerDisplayCaptureDeviceManager::createDisplayCaptureSource):
* Source/WebCore/platform/mediastream/gstreamer/GStreamerMediaStreamSource.cpp:
* Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCaptureSource.cpp:
(WebCore::GStreamerVideoCaptureSource::create):
* Source/WebCore/platform/mediastream/gstreamer/RealtimeIncomingSourceGStreamer.cpp:
(WebCore::RealtimeIncomingSourceGStreamer::unregisterClientLocked):
(WebCore::RealtimeIncomingSourceGStreamer::handleUpstreamEvent):
(WebCore::RealtimeIncomingSourceGStreamer::handleUpstreamQuery):
* Source/WebCore/platform/mediastream/gstreamer/RealtimeOutgoingAudioSourceGStreamer.cpp:
(WebCore::RealtimeOutgoingAudioSourceGStreamer::RealtimeOutgoingAudioSourceGStreamer):
* Source/WebCore/platform/mediastream/gstreamer/RealtimeOutgoingVideoSourceGStreamer.cpp:
(WebCore::RealtimeOutgoingVideoSourceGStreamer::RealtimeOutgoingVideoSourceGStreamer):
* Source/WebKit/NetworkProcess/BackgroundFetchLoad.cpp:
* Source/WebKit/NetworkProcess/Classifier/ITPThirdPartyDataForSpecificFirstParty.cpp:
(WebKit::ITPThirdPartyDataForSpecificFirstParty::toString const):
* Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp:
(WebKit::domainsToString):
(WebKit::buildList):
(WebKit::ResourceLoadStatisticsStore::debugLogDomainsInBatches):
(WebKit::ResourceLoadStatisticsStore::deleteTable):
(WebKit::ResourceLoadStatisticsStore::ensureAndMakeDomainList):
(WebKit::ResourceLoadStatisticsStore::insertDomainRelationshipList):
(WebKit::ResourceLoadStatisticsStore::incrementRecordsDeletedCountForDomains):
(WebKit::ResourceLoadStatisticsStore::markAsPrevalentIfHasRedirectedToPrevalent):
(WebKit::ResourceLoadStatisticsStore::findNotVeryPrevalentResources):
(WebKit::ResourceLoadStatisticsStore::grandfatherDataForDomains):
(WebKit::ResourceLoadStatisticsStore::logFrameNavigation):
(WebKit::ResourceLoadStatisticsStore::setDomainsAsPrevalent):
(WebKit::ResourceLoadStatisticsStore::updateCookieBlocking):
(WebKit::ResourceLoadStatisticsStore::clearGrandfathering):
(WebKit::ResourceLoadStatisticsStore::registrableDomainsToDeleteOrRestrictWebsiteDataFor):
(WebKit::ResourceLoadStatisticsStore::pruneStatisticsIfNeeded):
* Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.h:
* Source/WebKit/NetworkProcess/DatabaseUtilities.cpp:
(WebKit::insertDistinctValuesInTableStatement):
(WebKit::DatabaseUtilities::migrateDataToNewTablesIfNecessary):
(WebKit::DatabaseUtilities::columnsForTable):
(WebKit::DatabaseUtilities::addMissingColumnToTable):
* Source/WebKit/NetworkProcess/EarlyHintsResourceLoader.cpp:
(WebKit::EarlyHintsResourceLoader::startPreconnectTask):
* Source/WebKit/NetworkProcess/NetworkCORSPreflightChecker.cpp:
(WebKit::NetworkCORSPreflightChecker::willPerformHTTPRedirection):
* Source/WebKit/NetworkProcess/NetworkLoadChecker.cpp:
(WebKit::NetworkLoadChecker::checkRedirection):
(WebKit::NetworkLoadChecker::continueCheckingRequest):
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
(WebKit::NetworkProcess::newTestingSession):
* Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
(WebKit::NetworkResourceLoader::shouldInterruptLoadForCSPFrameAncestorsOrXFrameOptions):
(WebKit::NetworkResourceLoader::shouldInterruptNavigationForCrossOriginEmbedderPolicy):
(WebKit::NetworkResourceLoader::shouldInterruptWorkerLoadForCrossOriginEmbedderPolicy):
* Source/WebKit/NetworkProcess/cache/NetworkCache.cpp:
(WebKit::NetworkCache::Cache::dumpContentsToFile):
* Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
(-[WKNetworkSessionDelegate URLSession:task:didReceiveChallenge:completionHandler:]):
* Source/WebKit/NetworkProcess/mac/NetworkProcessMac.mm:
(WebKit::NetworkProcess::initializeSandbox):
* Source/WebKit/NetworkProcess/storage/BackgroundFetchStoreManager.cpp:
(WebKit::BackgroundFetchStoreManager::clearAllFetches):
(WebKit::BackgroundFetchStoreManager::storeFetchAfterQuotaCheck):
(WebKit::createFetchResponseBodyFile):
* Source/WebKit/NetworkProcess/storage/LocalStorageManager.cpp:
(WebKit::originToFileName):
* Source/WebKit/Platform/IPC/win/ConnectionWin.cpp:
(IPC::createServerAndClientIdentifiers):
* Source/WebKit/Platform/cocoa/WebPrivacyHelpers.mm:
(WebKit::LinkDecorationFilteringController::updateStrings):
(WebKit::requestLinkDecorationFilteringData):
(WebKit::configureForAdvancedPrivacyProtections):
* Source/WebKit/Scripts/PreferencesTemplates/WebPreferencesStoreDefaultsMap.cpp.erb:
* Source/WebKit/Shared/Cocoa/AuxiliaryProcessCocoa.mm:
(WebKit::AuxiliaryProcess::didReceiveInvalidMessage):
* Source/WebKit/Shared/EntryPointUtilities/Cocoa/XPCService/XPCServiceMain.mm:
(WebKit::checkFrameworkVersion):
* Source/WebKit/Shared/Gamepad/GamepadData.cpp:
(WebKit::GamepadData::loggingString const):
* Source/WebKit/Shared/WebBackForwardListItem.cpp:
(WebKit::WebBackForwardListItem::loggingString):
* Source/WebKit/Shared/WebMemorySampler.cpp:
(WebKit::WebMemorySampler::writeHeaders):
* Source/WebKit/Shared/mac/AuxiliaryProcessMac.mm:
(WebKit::setAndSerializeSandboxParameters):
* Source/WebKit/Shared/win/WebEventFactory.cpp:
(WebKit::keyIdentifierFromEvent):
* Source/WebKit/Shared/wpe/WebKeyboardEventWPE.cpp:
(WebKit::WebKeyboardEvent::keyIdentifierForWPEKeyval):
* Source/WebKit/UIProcess/API/APINavigation.cpp:
(API::Navigation::loggingString const):
* Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm:
(dumpCALayer):
* Source/WebKit/UIProcess/API/gtk/WebKitPrintOperation.cpp:
(webkitPrintOperationPreparePrint):
* Source/WebKit/UIProcess/API/ios/WKWebViewTestingIOS.mm:
(allowListedClassToString):
(dumpUIView):
* Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp:
(WebKit::WebAutomationSession::createBrowsingContext):
(WebKit::WebAutomationSession::setWindowFrameOfBrowsingContext):
(WebKit::fileCanBeAcceptedForUpload):
(WebKit::WebAutomationSession::resolveChildFrameHandle):
(WebKit::WebAutomationSession::computeElementLayout):
(WebKit::WebAutomationSession::setFilesToSelectForFileUpload):
(WebKit::WebAutomationSession::setFilesForInputFileUpload):
(WebKit::WebAutomationSession::addSingleCookie):
(WebKit::WebAutomationSession::setSessionPermissions):
(WebKit::WebAutomationSession::removeVirtualAuthenticator):
(WebKit::WebAutomationSession::addVirtualAuthenticatorCredential):
(WebKit::WebAutomationSession::removeVirtualAuthenticatorCredential):
(WebKit::WebAutomationSession::removeAllVirtualAuthenticatorCredentials):
(WebKit::WebAutomationSession::setVirtualAuthenticatorUserVerified):
(WebKit::WebAutomationSession::performMouseInteraction):
(WebKit::WebAutomationSession::performKeyboardInteractions):
(WebKit::WebAutomationSession::performInteractionSequence):
* Source/WebKit/UIProcess/Automation/WebAutomationSessionMacros.h:
* Source/WebKit/UIProcess/Cocoa/SOAuthorization/SubFrameSOAuthorizationSession.mm:
(WebKit::SubFrameSOAuthorizationSession::shouldInterruptLoadForCSPFrameAncestorsOrXFrameOptions):
* Source/WebKit/UIProcess/Cocoa/SafeBrowsingWarningCocoa.mm:
(WebKit::reportAnErrorURL):
(WebKit::malwareDetailsURL):
* Source/WebKit/UIProcess/Cocoa/WebPasteboardProxyCocoa.mm:
(WebKit::WebPasteboardProxy::testIPCSharedMemory):
* Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm:
(WebKit::WebExtensionContext::WebExtensionContext):
(WebKit::WebExtensionContext::load):
(WebKit::WebExtensionContext::setBaseURL):
* Source/WebKit/UIProcess/Inspector/WebInspectorUtilities.cpp:
(WebKit::defaultInspectorPageGroupIdentifierForPage):
* Source/WebKit/UIProcess/Inspector/glib/RemoteInspectorClient.cpp:
(WebKit::RemoteInspectorClient::setBackendCommands):
* Source/WebKit/UIProcess/Inspector/socket/RemoteInspectorClient.cpp:
(WebKit::RemoteInspectorClient::setBackendCommands):
* Source/WebKit/UIProcess/Inspector/socket/RemoteInspectorProtocolHandler.cpp:
(WebKit::RemoteInspectorProtocolHandler::updateTargetList):
* Source/WebKit/UIProcess/Inspector/win/WebInspectorUIProxyWin.cpp:
(WebKit::WebInspectorUIProxy::showSavePanelForSingleFile):
* Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp:
(WebKit::effectiveApplicationId):
* Source/WebKit/UIProcess/Launcher/glib/XDGDBusProxy.cpp:
(WebKit::XDGDBusProxy::dbusSessionProxy):
(WebKit::XDGDBusProxy::accessibilityProxy):
* Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp:
(WebKit::NetworkProcessProxy::requestStorageSpace):
* Source/WebKit/UIProcess/ProcessThrottler.cpp:
(WebKit::ProcessThrottler::assertionName const):
* Source/WebKit/UIProcess/SuspendedPageProxy.cpp:
(WebKit::SuspendedPageProxy::loggingString const):
* Source/WebKit/UIProcess/ViewGestureController.cpp:
(WebKit::ViewGestureController::SnapshotRemovalTracker::reset):
(WebKit::ViewGestureController::SnapshotRemovalTracker::stopWaitingForEvent):
(WebKit::ViewGestureController::SnapshotRemovalTracker::fireRemovalCallbackIfPossible):
(WebKit::ViewGestureController::SnapshotRemovalTracker::startWatchdog):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm:
(WebKit::LocalAuthenticator::makeCredential):
(WebKit::LocalAuthenticator::continueMakeCredentialAfterReceivingLAContext):
(WebKit::LocalAuthenticator::continueMakeCredentialAfterUserVerification):
(WebKit::LocalAuthenticator::continueGetAssertionAfterUserVerification):
* Source/WebKit/UIProcess/WebAuthentication/fido/U2fAuthenticator.cpp:
(WebKit::U2fAuthenticator::continueRegisterCommandAfterResponseReceived):
(WebKit::U2fAuthenticator::continueBogusCommandExcludeCredentialsMatchAfterResponseReceived):
(WebKit::U2fAuthenticator::continueBogusCommandNoCredentialsAfterResponseReceived):
* Source/WebKit/UIProcess/WebBackForwardList.cpp:
(WebKit::WebBackForwardList::loggingString):
* Source/WebKit/UIProcess/WebPageGroup.cpp:
(WebKit::pageGroupData):
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::loadServiceWorker):
* Source/WebKit/UIProcess/WebProcessPool.cpp:
(WebKit::WebProcessPool::processDidFinishLaunching):
(WebKit::WebProcessPool::startMemorySampler):
* Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm:
(WebKit::WebsiteDataStore::platformSetNetworkParameters):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::fetchDataAndApply):
* Source/WebKit/UIProcess/geoclue/GeoclueGeolocationProvider.cpp:
(WebKit::GeoclueGeolocationProvider::createPortalSession):
(WebKit::GeoclueGeolocationProvider::startPortalSession):
* Source/WebKit/UIProcess/win/WebPopupMenuProxyWin.cpp:
(WebKit::WebPopupMenuProxyWin::debugDescription const):
* Source/WebKit/WPEPlatform/wpe/wayland/WPEWaylandSHMPool.cpp:
(WPE::createSharedMemory):
* Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebPage.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundleScriptWorld.cpp:
(WebKit::uniqueWorldName):
* Source/WebKit/WebProcess/Inspector/WebPageInspectorTarget.cpp:
(WebKit::WebPageInspectorTarget::toTargetID):
* Source/WebKit/WebProcess/Model/mac/ARKitInlinePreviewModelPlayerMac.mm:
(WebKit::ARKitInlinePreviewModelPlayerMac::createPreviewsForModelWithURL):
* Source/WebKit/WebProcess/Network/WebResourceLoader.cpp:
(WebKit::WebResourceLoader::didFailServiceWorkerLoad):
* Source/WebKit/WebProcess/Plugins/PDF/PDFIncrementalLoader.mm:
(WebKit::PDFIncrementalLoader::receivedNonLinearizedPDFSentinel):
(WebKit::PDFIncrementalLoader::incrementalPDFStreamDidFinishLoading):
(WebKit::PDFIncrementalLoader::getResourceBytesAtPosition):
(WebKit::PDFIncrementalLoader::streamLoaderDidStart):
(WebKit::PDFIncrementalLoader::forgetStreamLoader):
(WebKit::PDFIncrementalLoader::requestDidCompleteWithBytes):
(WebKit::PDFIncrementalLoader::requestDidCompleteWithAccumulatedData):
(WebKit::PDFIncrementalLoader::dataProviderGetBytesAtPosition):
(WebKit::PDFIncrementalLoader::dataProviderGetByteRanges):
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm:
(WebKit::PDFPluginBase::startByteRangeRequest):
(WebKit::PDFPluginBase::debugDescription const):
* Source/WebKit/WebProcess/WebCoreSupport/WebResourceLoadObserver.cpp:
(WebKit::WebResourceLoadObserver::statisticsForURL):
* Source/WebKit/WebProcess/WebPage/Cocoa/WebCookieCacheCocoa.mm:
(WebKit::WebCookieCache::inMemoryStorageSession):
* Source/WebKit/WebProcess/WebPage/Cocoa/WebCookieJarCocoa.mm:
(WebKit::WebCookieJar::cookiesInPartitionedCookieStorage const):
* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
(WebKit::WebPage::getContentsAsString):
* Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm:
(WebKit::WebProcess::platformSetWebsiteDataStoreParameters):
(WebKit::WebProcess::initializeSandbox):
* Source/WebKit/webpushd/PushClientConnection.mm:
(WebPushD::PushClientConnection::setDebugModeIsEnabled):
(WebPushD::PushClientConnection::broadcastDebugMessage):
* Source/WebKit/webpushd/PushService.mm:
* Source/WebKitLegacy/mac/Scripts/PreferencesTemplates/WebPreferencesDefinitions.h.erb:
* Tools/TestWebKitAPI/Tests/IPC/IPCTestUtilities.cpp:
(TestWebKitAPI::ConnectionTestBase::teardownBase):
* Tools/TestWebKitAPI/Tests/WTF/StringBuilder.cpp:
(TestWebKitAPI::TEST(StringBuilderTest, VariadicAppend)):
* Tools/TestWebKitAPI/Tests/WTF/StringConcatenate.cpp:
(TestWebKitAPI::TEST(WTF, StringConcatenate)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Int)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Unsigned)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Float)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Double)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_FormattedDoubleFixedPrecision)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_FormattedDoubleFixedWidth)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Pad)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Tuple)):
* Tools/TestWebKitAPI/Tests/WTF/UUID.cpp:
(TEST(WTF, TestUUIDVersion4MakeString)):
* Tools/TestWebKitAPI/Tests/WebCore/ApplicationManifestParser.cpp:
(ApplicationManifestParserTest::parseTopLevelProperty):
(ApplicationManifestParserTest::parseIconFirstTopLevelProperty):
(ApplicationManifestParserTest::parseIconFirstTopLevelPropertyForSrc):
(ApplicationManifestParserTest::testScope):
(ApplicationManifestParserTest::testId):
* Tools/TestWebKitAPI/Tests/WebCore/DocumentOrder.cpp:
(TestWebKitAPI::string):
(TestWebKitAPI::typeStringSuffix):
(TestWebKitAPI::allPositionTypeFailures):
* Tools/TestWebKitAPI/Tests/WebCore/FileMonitor.cpp:
(TestWebKitAPI::createCommand):
(TestWebKitAPI::TEST_F):
* Tools/TestWebKitAPI/Tests/WebCore/HTTPHeaderField.cpp:
(canonicalizeHTTPHeader):
* Tools/TestWebKitAPI/Tests/WebCore/SecurityOrigin.cpp:
(TestWebKitAPI::TEST_F):
* Tools/TestWebKitAPI/Tests/WebCore/URLParserTextEncoding.cpp:
(TestWebKitAPI::TEST_F):
* Tools/WebKitTestRunner/cocoa/CrashReporterInfo.mm:
(WTR::setCrashReportApplicationSpecificInformationToURL):

Canonical link: <a href="https://commits.webkit.org/278972@main">https://commits.webkit.org/278972@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7e079d4da12794fc7e4434245482b38c3e197dea

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/52151 "26 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/31482 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/4571 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/55424 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/2865 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/54455 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/37880 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/2572 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/42439 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/1837 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/54247 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/29118 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/45007 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/23510 "Passed tests") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/51995 "Passed tests") | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/26386 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/2281 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/1033 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/45500 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/48291 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/2427 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/57021 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/51659 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/27276 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/2482 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/49836 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/28510 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/45118 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/49069 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/29422 "Built successfully") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/63966 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/7620 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/28254 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/12111 "Passed tests") | 
<!--EWS-Status-Bubble-End-->

darinadler commented on Source/WebKit/UIProcess/API/glib/WebKitAutomationSession.cpp:
```diff
@@ -431,9 +431,9 @@ String webkitAutomationSessionGetBrowserVersion(WebKitAutomationSession* session
+        return makeString(String::number(major), '.', String::number(minor));
 
-    return makeString(String::number(major), ".", String::number(minor), ".", String::number(micro));
+    return makeString(String::number(major), '.', String::number(minor), '.', String::number(micro));
```
Don’t need `String::number` here.

darinadler commented on Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp:
```diff
@@ -422,12 +422,12 @@ void WebKitProtocolHandler::handleGPU(WebKitURISchemeRequest* request)
 
 #if USE(LIBDRM)
-    if (strcmp(policy, "never")) {
+    if (strcmp(policy.characters(), "never")) {
```
Kind of wish we had an `!=` we could use here instead.

cdumez replied:
Actually, I believe we do for ASCIILiteral. I'll update.

darinadler commented on Source/WebKit/UIProcess/Cocoa/NavigationState.mm:
```diff
@@ -429,7 +429,7 @@ static void interceptMarketplaceKitNavigation(Ref<API::NavigationAction>&& actio
             return;
 
-        weakPage->addConsoleMessage(*sourceFrameID, MessageSource::Network, MessageLevel::Error, makeString("Can't handle MarketplaceKit link ", url.string().utf8().data(), " due to error: "_s, error));
+        weakPage->addConsoleMessage(*sourceFrameID, MessageSource::Network, MessageLevel::Error, makeString("Can't handle MarketplaceKit link "_s, url.string().utf8().span(), " due to error: "_s, error));
```
I don’t think we should convert this to UTF-8 and back to Latin-1/UTF-16, unless there is some benefit to that I am missing. I think we can just write `url.string()` instead of `url.string().utf8().span()`.

darinadler commented on Source/WebKit/UIProcess/WebBackForwardList.cpp:
```diff
@@ -549,15 +549,15 @@ String WebBackForwardList::loggingString()
     }
 
-    return makeString("\n", builder.toString());
+    return makeString('\n', builder.toString());
```
Lets put the `'\n'` into the `StringBuilder` at the start of the function and avoid this expensive prepend operation.

darinadler commented on Source/WebKit/UIProcess/WebProcessPool.cpp:
```diff
@@ -1097,7 +1097,7 @@ void WebProcessPool::processDidFinishLaunching(WebProcessProxy& process)
         SandboxExtension::Handle sampleLogSandboxHandle;        
         WallTime now = WallTime::now();
-        auto sampleLogFilePath = makeString("WebProcess", static_cast<unsigned long long>(now.secondsSinceEpoch().seconds()), "pid", process.processID());
+        auto sampleLogFilePath = makeString("WebProcess"_s, static_cast<unsigned long long>(now.secondsSinceEpoch().seconds()), "pid"_s, process.processID());
```
I wonder if we could write this as `now.secondsSinceEpoch().secondsAs<uint64_t>()` instead?

darinadler commented on Source/WebKit/UIProcess/WebProcessPool.cpp:
```diff
@@ -1617,7 +1617,7 @@ void WebProcessPool::startMemorySampler(const double interval)
         SandboxExtension::Handle sampleLogSandboxHandle;
         WallTime now = WallTime::now();
-        auto sampleLogFilePath = makeString("WebProcess", static_cast<unsigned long long>(now.secondsSinceEpoch().seconds()));
+        auto sampleLogFilePath = makeString("WebProcess"_s, static_cast<unsigned long long>(now.secondsSinceEpoch().seconds()));
```
I wonder if we could write this as `now.secondsSinceEpoch().secondsAs<uint64_t>()` instead?

darinadler commented on Source/WebKit/webpushd/PushClientConnection.mm:
```diff
@@ -170,17 +170,17 @@
+        messageIdentifier = makeString("[(0x"_s, hex(reinterpret_cast<uint64_t>(m_xpcConnection.get()), WTF::HexConversionMode::Lowercase), ") ("_s, String::number(identifier().toUInt64()), " )] "_s);
     else
-        messageIdentifier = makeString("[", signingIdentifier, " (", String::number(identifier().toUInt64()), ")] ");
+        messageIdentifier = makeString('[', signingIdentifier, " ("_s, String::number(identifier().toUInt64()), ")] "_s);
```
Unnecessary use of `String::number` here.

darinadler commented on Source/WebKit/webpushd/PushClientConnection.mm:
```diff
@@ -170,17 +170,17 @@
     auto signingIdentifier = hostAppCodeSigningIdentifier();
     if (signingIdentifier.isEmpty())
-        messageIdentifier = makeString("[(0x", hex(reinterpret_cast<uint64_t>(m_xpcConnection.get()), WTF::HexConversionMode::Lowercase), ") (", String::number(identifier().toUInt64()), " )] ");
+        messageIdentifier = makeString("[(0x"_s, hex(reinterpret_cast<uint64_t>(m_xpcConnection.get()), WTF::HexConversionMode::Lowercase), ") ("_s, String::number(identifier().toUInt64()), " )] "_s);
```
Unnecessary use of `String::number` here.

darinadler commented on Tools/DumpRenderTree/mac/DumpRenderTree.mm:
```diff
@@ -1948,7 +1948,7 @@ static void runTest(const std::string& inputLine)
         testPath = [url absoluteString];
 
-    auto message = makeString("CRASHING TEST: ", testPath.UTF8String);
+    auto message = makeString("CRASHING TEST: "_s, span(testPath.UTF8String));
```
Can we just pass in `testPath` and let the `NSString` capabilities of `makeString` take care of it, instead of calling `UTF8String`? In fact, this existing code seems like it will reinterpret UTF-8 characters as Latin-1 and then convert back to UTF-8, so that might even fix a non-ASCII characters bug.

cdumez replied:
```
/Volumes/Data/worker/macOS-Sonoma-Debug-Build-EWS/build/WebKitBuild/Debug/usr/local/include/wtf/text/StringConcatenate.h:534:38: error: implicit instantiation of undefined template 'WTF::StringTypeAdapter<NSString *>'
```
Doesn't seem like makeString() plays nicely with NSString* ?

cdumez replied:
Oh, I'll try to include StringConcatenateCF.h

darinadler commented on Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp:
```diff
@@ -300,7 +300,7 @@ static String attributesOfElement(AccessibilityUIElement& element)
     String description = element.description()->string();
     if (!description.isEmpty())
-        builder.append(description.utf8().data(), '\n');
+        builder.append(description.utf8().span(), '\n');
```
How about just `description` instead of `description.utf8().span()`? The existing code seems to convert to UTF-8 then back as Latin-1, for no good reason.

darinadler commented on Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp:
```diff
@@ -905,29 +905,29 @@ JSRetainPtr<JSStringRef> AccessibilityUIElement::computedRoleString()
 {
     m_element->updateBackingStore();
-    auto titleValue = makeString("AXTitle: ", String::fromUTF8(m_element->name().span()));
+    auto titleValue = makeString("AXTitle: "_s, String::fromUTF8(m_element->name().span()));
```
No need to use `String::fromUTF8` here. Very confusing!

darinadler commented on Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp:
```diff
@@ -905,29 +905,29 @@ JSRetainPtr<JSStringRef> AccessibilityUIElement::computedRoleString()
 {
     m_element->updateBackingStore();
-    auto descriptionValue = makeString("AXDescription: ", String::fromUTF8(m_element->description().span()));
+    auto descriptionValue = makeString("AXDescription: "_s, String::fromUTF8(m_element->description().span()));
```
No need to use `String::fromUTF8` here. Very confusing!

darinadler commented on Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp:
```diff
@@ -953,14 +953,14 @@ JSRetainPtr<JSStringRef> AccessibilityUIElement::stringValue()
         if (auto menu = childAtIndex(0)) {
             if (auto* selectedChild = menu->m_element->selectedChild(0))
-                return OpaqueJSString::tryCreate(makeString("AXValue: ", String::fromUTF8(selectedChild->name().span()))).leakRef();
+                return OpaqueJSString::tryCreate(makeString("AXValue: "_s, String::fromUTF8(selectedChild->name().span()))).leakRef();
```
No need to use `String::fromUTF8` here. Very confusing!

cdumez committed:
Stop using `const char*` in string concatenations
https://bugs.webkit.org/show_bug.cgi?id=274297

Reviewed by Darin Adler.

* Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml:
* Source/WTF/wtf/text/StringConcatenate.h:
* Source/WebCore/platform/mediastream/gstreamer/GStreamerAudioCaptureSource.cpp:
(WebCore::GStreamerAudioCaptureSource::create):
* Source/WebCore/platform/mediastream/gstreamer/GStreamerDisplayCaptureDeviceManager.cpp:
(WebCore::GStreamerDisplayCaptureDeviceManager::computeCaptureDevices):
(WebCore::GStreamerDisplayCaptureDeviceManager::createDisplayCaptureSource):
* Source/WebCore/platform/mediastream/gstreamer/GStreamerMediaStreamSource.cpp:
* Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCaptureSource.cpp:
(WebCore::GStreamerVideoCaptureSource::create):
* Source/WebCore/platform/mediastream/gstreamer/RealtimeIncomingSourceGStreamer.cpp:
(WebCore::RealtimeIncomingSourceGStreamer::unregisterClientLocked):
(WebCore::RealtimeIncomingSourceGStreamer::handleUpstreamEvent):
(WebCore::RealtimeIncomingSourceGStreamer::handleUpstreamQuery):
* Source/WebCore/platform/mediastream/gstreamer/RealtimeOutgoingAudioSourceGStreamer.cpp:
(WebCore::RealtimeOutgoingAudioSourceGStreamer::RealtimeOutgoingAudioSourceGStreamer):
* Source/WebCore/platform/mediastream/gstreamer/RealtimeOutgoingVideoSourceGStreamer.cpp:
(WebCore::RealtimeOutgoingVideoSourceGStreamer::RealtimeOutgoingVideoSourceGStreamer):
* Source/WebKit/NetworkProcess/BackgroundFetchLoad.cpp:
* Source/WebKit/NetworkProcess/Classifier/ITPThirdPartyDataForSpecificFirstParty.cpp:
(WebKit::ITPThirdPartyDataForSpecificFirstParty::toString const):
* Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp:
(WebKit::domainsToString):
(WebKit::buildList):
(WebKit::ResourceLoadStatisticsStore::debugLogDomainsInBatches):
(WebKit::ResourceLoadStatisticsStore::deleteTable):
(WebKit::ResourceLoadStatisticsStore::ensureAndMakeDomainList):
(WebKit::ResourceLoadStatisticsStore::insertDomainRelationshipList):
(WebKit::ResourceLoadStatisticsStore::incrementRecordsDeletedCountForDomains):
(WebKit::ResourceLoadStatisticsStore::markAsPrevalentIfHasRedirectedToPrevalent):
(WebKit::ResourceLoadStatisticsStore::findNotVeryPrevalentResources):
(WebKit::ResourceLoadStatisticsStore::grandfatherDataForDomains):
(WebKit::ResourceLoadStatisticsStore::logFrameNavigation):
(WebKit::ResourceLoadStatisticsStore::setDomainsAsPrevalent):
(WebKit::ResourceLoadStatisticsStore::updateCookieBlocking):
(WebKit::ResourceLoadStatisticsStore::clearGrandfathering):
(WebKit::ResourceLoadStatisticsStore::registrableDomainsToDeleteOrRestrictWebsiteDataFor):
(WebKit::ResourceLoadStatisticsStore::pruneStatisticsIfNeeded):
* Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.h:
* Source/WebKit/NetworkProcess/DatabaseUtilities.cpp:
(WebKit::insertDistinctValuesInTableStatement):
(WebKit::DatabaseUtilities::migrateDataToNewTablesIfNecessary):
(WebKit::DatabaseUtilities::columnsForTable):
(WebKit::DatabaseUtilities::addMissingColumnToTable):
* Source/WebKit/NetworkProcess/EarlyHintsResourceLoader.cpp:
(WebKit::EarlyHintsResourceLoader::startPreconnectTask):
* Source/WebKit/NetworkProcess/NetworkCORSPreflightChecker.cpp:
(WebKit::NetworkCORSPreflightChecker::willPerformHTTPRedirection):
* Source/WebKit/NetworkProcess/NetworkLoadChecker.cpp:
(WebKit::NetworkLoadChecker::checkRedirection):
(WebKit::NetworkLoadChecker::continueCheckingRequest):
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
(WebKit::NetworkProcess::newTestingSession):
* Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
(WebKit::NetworkResourceLoader::shouldInterruptLoadForCSPFrameAncestorsOrXFrameOptions):
(WebKit::NetworkResourceLoader::shouldInterruptNavigationForCrossOriginEmbedderPolicy):
(WebKit::NetworkResourceLoader::shouldInterruptWorkerLoadForCrossOriginEmbedderPolicy):
* Source/WebKit/NetworkProcess/cache/NetworkCache.cpp:
(WebKit::NetworkCache::Cache::dumpContentsToFile):
* Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
(-[WKNetworkSessionDelegate URLSession:task:didReceiveChallenge:completionHandler:]):
* Source/WebKit/NetworkProcess/mac/NetworkProcessMac.mm:
(WebKit::NetworkProcess::initializeSandbox):
* Source/WebKit/NetworkProcess/storage/BackgroundFetchStoreManager.cpp:
(WebKit::BackgroundFetchStoreManager::clearAllFetches):
(WebKit::BackgroundFetchStoreManager::storeFetchAfterQuotaCheck):
(WebKit::createFetchResponseBodyFile):
* Source/WebKit/NetworkProcess/storage/LocalStorageManager.cpp:
(WebKit::originToFileName):
* Source/WebKit/Platform/IPC/win/ConnectionWin.cpp:
(IPC::createServerAndClientIdentifiers):
* Source/WebKit/Platform/cocoa/WebPrivacyHelpers.mm:
(WebKit::LinkDecorationFilteringController::updateStrings):
(WebKit::requestLinkDecorationFilteringData):
(WebKit::configureForAdvancedPrivacyProtections):
* Source/WebKit/Scripts/PreferencesTemplates/WebPreferencesStoreDefaultsMap.cpp.erb:
* Source/WebKit/Shared/Cocoa/AuxiliaryProcessCocoa.mm:
(WebKit::AuxiliaryProcess::didReceiveInvalidMessage):
* Source/WebKit/Shared/EntryPointUtilities/Cocoa/XPCService/XPCServiceMain.mm:
(WebKit::checkFrameworkVersion):
* Source/WebKit/Shared/Gamepad/GamepadData.cpp:
(WebKit::GamepadData::loggingString const):
* Source/WebKit/Shared/WebBackForwardListItem.cpp:
(WebKit::WebBackForwardListItem::loggingString):
* Source/WebKit/Shared/WebMemorySampler.cpp:
(WebKit::WebMemorySampler::writeHeaders):
* Source/WebKit/Shared/mac/AuxiliaryProcessMac.mm:
(WebKit::setAndSerializeSandboxParameters):
* Source/WebKit/Shared/win/WebEventFactory.cpp:
(WebKit::keyIdentifierFromEvent):
* Source/WebKit/Shared/wpe/WebKeyboardEventWPE.cpp:
(WebKit::WebKeyboardEvent::keyIdentifierForWPEKeyval):
* Source/WebKit/UIProcess/API/APINavigation.cpp:
(API::Navigation::loggingString const):
* Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm:
(dumpCALayer):
* Source/WebKit/UIProcess/API/gtk/WebKitPrintOperation.cpp:
(webkitPrintOperationPreparePrint):
* Source/WebKit/UIProcess/API/ios/WKWebViewTestingIOS.mm:
(allowListedClassToString):
(dumpUIView):
* Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp:
(WebKit::WebAutomationSession::createBrowsingContext):
(WebKit::WebAutomationSession::setWindowFrameOfBrowsingContext):
(WebKit::fileCanBeAcceptedForUpload):
(WebKit::WebAutomationSession::resolveChildFrameHandle):
(WebKit::WebAutomationSession::computeElementLayout):
(WebKit::WebAutomationSession::setFilesToSelectForFileUpload):
(WebKit::WebAutomationSession::setFilesForInputFileUpload):
(WebKit::WebAutomationSession::addSingleCookie):
(WebKit::WebAutomationSession::setSessionPermissions):
(WebKit::WebAutomationSession::removeVirtualAuthenticator):
(WebKit::WebAutomationSession::addVirtualAuthenticatorCredential):
(WebKit::WebAutomationSession::removeVirtualAuthenticatorCredential):
(WebKit::WebAutomationSession::removeAllVirtualAuthenticatorCredentials):
(WebKit::WebAutomationSession::setVirtualAuthenticatorUserVerified):
(WebKit::WebAutomationSession::performMouseInteraction):
(WebKit::WebAutomationSession::performKeyboardInteractions):
(WebKit::WebAutomationSession::performInteractionSequence):
* Source/WebKit/UIProcess/Automation/WebAutomationSessionMacros.h:
* Source/WebKit/UIProcess/Cocoa/SOAuthorization/SubFrameSOAuthorizationSession.mm:
(WebKit::SubFrameSOAuthorizationSession::shouldInterruptLoadForCSPFrameAncestorsOrXFrameOptions):
* Source/WebKit/UIProcess/Cocoa/SafeBrowsingWarningCocoa.mm:
(WebKit::reportAnErrorURL):
(WebKit::malwareDetailsURL):
* Source/WebKit/UIProcess/Cocoa/WebPasteboardProxyCocoa.mm:
(WebKit::WebPasteboardProxy::testIPCSharedMemory):
* Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm:
(WebKit::WebExtensionContext::WebExtensionContext):
(WebKit::WebExtensionContext::load):
(WebKit::WebExtensionContext::setBaseURL):
* Source/WebKit/UIProcess/Inspector/WebInspectorUtilities.cpp:
(WebKit::defaultInspectorPageGroupIdentifierForPage):
* Source/WebKit/UIProcess/Inspector/glib/RemoteInspectorClient.cpp:
(WebKit::RemoteInspectorClient::setBackendCommands):
* Source/WebKit/UIProcess/Inspector/socket/RemoteInspectorClient.cpp:
(WebKit::RemoteInspectorClient::setBackendCommands):
* Source/WebKit/UIProcess/Inspector/socket/RemoteInspectorProtocolHandler.cpp:
(WebKit::RemoteInspectorProtocolHandler::updateTargetList):
* Source/WebKit/UIProcess/Inspector/win/WebInspectorUIProxyWin.cpp:
(WebKit::WebInspectorUIProxy::showSavePanelForSingleFile):
* Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp:
(WebKit::effectiveApplicationId):
* Source/WebKit/UIProcess/Launcher/glib/XDGDBusProxy.cpp:
(WebKit::XDGDBusProxy::dbusSessionProxy):
(WebKit::XDGDBusProxy::accessibilityProxy):
* Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp:
(WebKit::NetworkProcessProxy::requestStorageSpace):
* Source/WebKit/UIProcess/ProcessThrottler.cpp:
(WebKit::ProcessThrottler::assertionName const):
* Source/WebKit/UIProcess/SuspendedPageProxy.cpp:
(WebKit::SuspendedPageProxy::loggingString const):
* Source/WebKit/UIProcess/ViewGestureController.cpp:
(WebKit::ViewGestureController::SnapshotRemovalTracker::reset):
(WebKit::ViewGestureController::SnapshotRemovalTracker::stopWaitingForEvent):
(WebKit::ViewGestureController::SnapshotRemovalTracker::fireRemovalCallbackIfPossible):
(WebKit::ViewGestureController::SnapshotRemovalTracker::startWatchdog):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm:
(WebKit::LocalAuthenticator::makeCredential):
(WebKit::LocalAuthenticator::continueMakeCredentialAfterReceivingLAContext):
(WebKit::LocalAuthenticator::continueMakeCredentialAfterUserVerification):
(WebKit::LocalAuthenticator::continueGetAssertionAfterUserVerification):
* Source/WebKit/UIProcess/WebAuthentication/fido/U2fAuthenticator.cpp:
(WebKit::U2fAuthenticator::continueRegisterCommandAfterResponseReceived):
(WebKit::U2fAuthenticator::continueBogusCommandExcludeCredentialsMatchAfterResponseReceived):
(WebKit::U2fAuthenticator::continueBogusCommandNoCredentialsAfterResponseReceived):
* Source/WebKit/UIProcess/WebBackForwardList.cpp:
(WebKit::WebBackForwardList::loggingString):
* Source/WebKit/UIProcess/WebPageGroup.cpp:
(WebKit::pageGroupData):
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::loadServiceWorker):
* Source/WebKit/UIProcess/WebProcessPool.cpp:
(WebKit::WebProcessPool::processDidFinishLaunching):
(WebKit::WebProcessPool::startMemorySampler):
* Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm:
(WebKit::WebsiteDataStore::platformSetNetworkParameters):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::fetchDataAndApply):
* Source/WebKit/UIProcess/geoclue/GeoclueGeolocationProvider.cpp:
(WebKit::GeoclueGeolocationProvider::createPortalSession):
(WebKit::GeoclueGeolocationProvider::startPortalSession):
* Source/WebKit/UIProcess/win/WebPopupMenuProxyWin.cpp:
(WebKit::WebPopupMenuProxyWin::debugDescription const):
* Source/WebKit/WPEPlatform/wpe/wayland/WPEWaylandSHMPool.cpp:
(WPE::createSharedMemory):
* Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebPage.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundleScriptWorld.cpp:
(WebKit::uniqueWorldName):
* Source/WebKit/WebProcess/Inspector/WebPageInspectorTarget.cpp:
(WebKit::WebPageInspectorTarget::toTargetID):
* Source/WebKit/WebProcess/Model/mac/ARKitInlinePreviewModelPlayerMac.mm:
(WebKit::ARKitInlinePreviewModelPlayerMac::createPreviewsForModelWithURL):
* Source/WebKit/WebProcess/Network/WebResourceLoader.cpp:
(WebKit::WebResourceLoader::didFailServiceWorkerLoad):
* Source/WebKit/WebProcess/Plugins/PDF/PDFIncrementalLoader.mm:
(WebKit::PDFIncrementalLoader::receivedNonLinearizedPDFSentinel):
(WebKit::PDFIncrementalLoader::incrementalPDFStreamDidFinishLoading):
(WebKit::PDFIncrementalLoader::getResourceBytesAtPosition):
(WebKit::PDFIncrementalLoader::streamLoaderDidStart):
(WebKit::PDFIncrementalLoader::forgetStreamLoader):
(WebKit::PDFIncrementalLoader::requestDidCompleteWithBytes):
(WebKit::PDFIncrementalLoader::requestDidCompleteWithAccumulatedData):
(WebKit::PDFIncrementalLoader::dataProviderGetBytesAtPosition):
(WebKit::PDFIncrementalLoader::dataProviderGetByteRanges):
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm:
(WebKit::PDFPluginBase::startByteRangeRequest):
(WebKit::PDFPluginBase::debugDescription const):
* Source/WebKit/WebProcess/WebCoreSupport/WebResourceLoadObserver.cpp:
(WebKit::WebResourceLoadObserver::statisticsForURL):
* Source/WebKit/WebProcess/WebPage/Cocoa/WebCookieCacheCocoa.mm:
(WebKit::WebCookieCache::inMemoryStorageSession):
* Source/WebKit/WebProcess/WebPage/Cocoa/WebCookieJarCocoa.mm:
(WebKit::WebCookieJar::cookiesInPartitionedCookieStorage const):
* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
(WebKit::WebPage::getContentsAsString):
* Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm:
(WebKit::WebProcess::platformSetWebsiteDataStoreParameters):
(WebKit::WebProcess::initializeSandbox):
* Source/WebKit/webpushd/PushClientConnection.mm:
(WebPushD::PushClientConnection::setDebugModeIsEnabled):
(WebPushD::PushClientConnection::broadcastDebugMessage):
* Source/WebKit/webpushd/PushService.mm:
* Source/WebKitLegacy/mac/Scripts/PreferencesTemplates/WebPreferencesDefinitions.h.erb:
* Tools/TestWebKitAPI/Tests/IPC/IPCTestUtilities.cpp:
(TestWebKitAPI::ConnectionTestBase::teardownBase):
* Tools/TestWebKitAPI/Tests/WTF/StringBuilder.cpp:
(TestWebKitAPI::TEST(StringBuilderTest, VariadicAppend)):
* Tools/TestWebKitAPI/Tests/WTF/StringConcatenate.cpp:
(TestWebKitAPI::TEST(WTF, StringConcatenate)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Int)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Unsigned)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Float)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Double)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_FormattedDoubleFixedPrecision)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_FormattedDoubleFixedWidth)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Pad)):
(TestWebKitAPI::TEST(WTF, StringConcatenate_Tuple)):
* Tools/TestWebKitAPI/Tests/WTF/UUID.cpp:
(TEST(WTF, TestUUIDVersion4MakeString)):
* Tools/TestWebKitAPI/Tests/WebCore/ApplicationManifestParser.cpp:
(ApplicationManifestParserTest::parseTopLevelProperty):
(ApplicationManifestParserTest::parseIconFirstTopLevelProperty):
(ApplicationManifestParserTest::parseIconFirstTopLevelPropertyForSrc):
(ApplicationManifestParserTest::testScope):
(ApplicationManifestParserTest::testId):
* Tools/TestWebKitAPI/Tests/WebCore/DocumentOrder.cpp:
(TestWebKitAPI::string):
(TestWebKitAPI::typeStringSuffix):
(TestWebKitAPI::allPositionTypeFailures):
* Tools/TestWebKitAPI/Tests/WebCore/FileMonitor.cpp:
(TestWebKitAPI::createCommand):
(TestWebKitAPI::TEST_F):
* Tools/TestWebKitAPI/Tests/WebCore/HTTPHeaderField.cpp:
(canonicalizeHTTPHeader):
* Tools/TestWebKitAPI/Tests/WebCore/SecurityOrigin.cpp:
(TestWebKitAPI::TEST_F):
* Tools/TestWebKitAPI/Tests/WebCore/URLParserTextEncoding.cpp:
(TestWebKitAPI::TEST_F):
* Tools/WebKitTestRunner/cocoa/CrashReporterInfo.mm:
(WTR::setCrashReportApplicationSpecificInformationToURL):

Canonical link: https://commits.webkit.org/278972@main
